Modules/ThirdParty/Expat/src/expat/itk_expat_mangle.h

```diff
@@ -27,79 +27,96 @@ ITK.
-#define XML_ParserCreateNS itk_expat_XML_ParserCreateNS
-#define XML_ParserFree itk_expat_XML_ParserFree
-#define XmlParseXmlDecl itk_expat_XmlParseXmlDecl
-#define XmlParseXmlDeclNS itk_expat_XmlParseXmlDeclNS
-#define XmlPrologStateInit itk_expat_XmlPrologStateInit
-#define XmlPrologStateInitExternalEntity itk_expat_XmlPrologStateInitExternalEntity
-#define XML_SetAttlistDeclHandler itk_expat_XML_SetAttlistDeclHandler
-#define XML_SetBase itk_expat_XML_SetBase
-#define XML_SetCdataSectionHandler itk_expat_XML_SetCdataSectionHandler
-#define XML_SetCharacterDataHandler itk_expat_XML_SetCharacterDataHandler
-#define XML_SetCommentHandler itk_expat_XML_SetCommentHandler
-#define XML_SetDefaultHandler itk_expat_XML_SetDefaultHandler
-#define XML_SetDefaultHandlerExpand itk_expat_XML_SetDefaultHandlerExpand
-#define XML_SetDoctypeDeclHandler itk_expat_XML_SetDoctypeDeclHandler
-#define XML_SetElementDeclHandler itk_expat_XML_SetElementDeclHandler
-#define XML_SetElementHandler itk_expat_XML_SetElementHandler
-#define XML_SetEncoding itk_expat_XML_SetEncoding
-#define XML_SetEndCdataSectionHandler itk_expat_XML_SetEndCdataSectionHandler
-#define XML_SetEndDoctypeDeclHandler itk_expat_XML_SetEndDoctypeDeclHandler
-#define XML_SetEndElementHandler itk_expat_XML_SetEndElementHandler
-#define XML_SetEndNamespaceDeclHandler itk_expat_XML_SetEndNamespaceDeclHandler
-#define XML_SetEntityDeclHandler itk_expat_XML_SetEntityDeclHandler
-#define XML_SetExternalEntityRefHandler itk_expat_XML_SetExternalEntityRefHandler
-#define XML_SetExternalEntityRefHandlerArg itk_expat_XML_SetExternalEntityRefHandlerArg
-#define XML_SetNamespaceDeclHandler itk_expat_XML_SetNamespaceDeclHandler
-#define XML_SetNotationDeclHandler itk_expat_XML_SetNotationDeclHandler
-#define XML_SetNotStandaloneHandler itk_expat_XML_SetNotStandaloneHandler
-#define XML_SetParamEntityParsing itk_expat_XML_SetParamEntityParsing
-#define XML_SetProcessingInstructionHandler itk_expat_XML_SetProcessingInstructionHandler
-#define XML_SetReturnNSTriplet itk_expat_XML_SetReturnNSTriplet
-#define XML_SetStartCdataSectionHandler itk_expat_XML_SetStartCdataSectionHandler
-#define XML_SetStartDoctypeDeclHandler itk_expat_XML_SetStartDoctypeDeclHandler
-#define XML_SetStartElementHandler itk_expat_XML_SetStartElementHandler
-#define XML_SetStartNamespaceDeclHandler itk_expat_XML_SetStartNamespaceDeclHandler
-#define XML_SetUnknownEncodingHandler itk_expat_XML_SetUnknownEncodingHandler
-#define XML_SetUnparsedEntityDeclHandler itk_expat_XML_SetUnparsedEntityDeclHandler
-#define XML_SetUserData itk_expat_XML_SetUserData
-#define XML_SetXmlDeclHandler itk_expat_XML_SetXmlDeclHandler
-#define XmlSizeOfUnknownEncoding itk_expat_XmlSizeOfUnknownEncoding
-#define XML_UseParserAsHandlerArg itk_expat_XML_UseParserAsHandlerArg
-#define XmlUtf16Encode itk_expat_XmlUtf16Encode
-#define XmlUtf8Encode itk_expat_XmlUtf8Encode
+#define _XML_DefaultCurrent itk_expat__XML_DefaultCurrent
```

Comment:
All symbols now having underscore prefix? That seems suspicious. See whether removing them fixes the Windows build failure. Example:
```suggestion
#define XML_DefaultCurrent itk_expat__XML_DefaultCurrent
```